code/modules/ruins/objects_and_mobs/sin_ruins.dm

```diff
@@ -16,6 +16,7 @@
 		return
 	obj_flags |= IN_USE
 	user.adjustCloneLoss(20)
+	user.adjustBrainLoss(5)
```

Comment:
Here's code that works:
```if(isIPC(user))
		user.adjustBrainLoss(20)
	else
		user.adjustCloneLoss(20)
```

Funny thing is, when I tested it, I won on the first try. Go figure.

Reply:
I ROLLED THE DIE DIRECTLY AFTERWARDS AND ROLLED A 20 CMON WHY DOES THIS HAPPEN NOW?!

Reply:
Mate, if only I knew what to write as the if() parameter in the first place.... I have zero knowledge on how race checks should be done properly.

Reply:
I had just seen the param in another PR, and thought it might work. Just so happens it does.

Reply:
er, wait a moment, I never sent the if() param, I'm an idiot

Reply:
Uh, whoops, I messed up the markup in my comment. If param is in there now, sorry

Reply:
Thanks, mate